tests/psutil/test.q

```diff
@@ -1,4 +1,4 @@
-.finos.dep.include"psutil/psutil.q"
+.finos.dep.include"../../q/psutil/psutil.q"
```

Comment:
Since tests are looking at the code from the outside, this should probably be ```.finos.dep.loadScriptIn["finos/kdb";"psutil/psutil.q"```

Reply:
How's it look now?

Reply:
That's what it should look like.